features/type_users.feature

```diff
@@ -93,7 +93,7 @@ Scenario: User can submit a create form that includes user type and SID for Stud
     | User Name         | Fakeuser              |
     | Email             | fakeuser@berkeley.edu |
     | Preferred Contact | 555-555-5555          |
-    | Github Uid        | fakegithubuid         |
+    | GitHub Username   | fakegithubuid         |
```

Comment:
Already included in previous PR, but to pass CI, we have to include modified tests.

Please ignore this file.